flake8_trio/visitors/helpers.py

```diff
@@ -103,10 +103,10 @@ def has_exception(node: ast.expr | None) -> str:
         if (
             isinstance(node, ast.Attribute)
             and isinstance(node.value, ast.Name)
-            and node.value.id == "trio"
+            and node.value.id in ("trio", "anyio")
             and node.attr == "Cancelled"
         ):
-            return "trio.Cancelled"
+            return f"{node.value.id}.Cancelled"
```

Comment:
Unfortunately `anyio` has a *slightly* different API here; see [`anyio.get_cancelled_exc_class()`](https://anyio.readthedocs.io/en/stable/api.html#anyio.get_cancelled_exc_class).  That's important because you can have both `trio` and `asyncio` in the same program (ugh, but sometimes the least-bad option!).

We probably also want to lint for "no assigning that result because it breaks our linter, as well as multi-backend programs".

Reply:
I assume you mean `anyio` and not `asyncio`? Maybe not the best name they could've chosen :sweat_smile: 

I'll probably push this to a separate issue+PR.

Reply:
:scream:

Reply:
Tracked in #122